src/build.rs

```diff
@@ -28,6 +28,10 @@ fn main() {
         .unwrap()
         .write_all(docker_images().as_bytes())
         .unwrap();
+
+    if env::var("NIX_PATH").is_ok() {
+        println!("cargo:rustc-cfg=cross_nixos");
+    }
```

Comment:
In Nix when using Flake, this is not set.

Reply:
Any other envvar that will be set? I tried using the base system for the Docker image, which is missing a lot of things. I can try checking if one of a subset of envvars are set, short-circuiting once the first is.

Reply:
In the base Docker image for NixOS, the only two NIX-specific envvars set are:
- `NIX_PATH`
- `NIX_SSL_CERT_FILE`

I don't the latter is reliable. `uname` gets the host data for the container, so it's also not reliable. There is no `/etc/os-release` either here. Is checking for `/etc/nix/nix.conf` reliable? If not I'll just change it back to `#[ignored]`.